plugins/quickinspector/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ if(Qt5Quick_FOUND AND HAVE_PRIVATE_Qt5Quick_HEADERS AND NOT ${Qt5Quick_VERSION}
   include_directories(SYSTEM ${Qt5Quick_PRIVATE_INCLUDE_DIRS})
 
   add_library(gammaray_quickinspector_shared STATIC ${gammaray_quickinspector_shared_srcs})
-  target_link_libraries(gammaray_quickinspector_shared gammaray_common Qt5::Gui Qt5::Quick)
+  target_link_libraries(gammaray_quickinspector_shared gammaray_common gammaray_core Qt5::Gui Qt5::Quick)
```

Comment:
This links probe code into the client, that is dangerous as that makes assumptions of running in target everywhere. Can we avoid this?